packages/components/src/mobile/link-settings/style.native.scss

```diff
@@ -2,3 +2,20 @@
 	padding-left: 0;
 	padding-right: 0;
 }
+
+// used in both light and dark modes
+.placeholderTextColor {
+	color: #87a6bc;
+}
+
+.optionIcon {
+	color: $blue-50;
+}
+
+.optionIconDark {
+	color: $blue-30;
+}
+
+.unselectedOptionIcon {
+	opacity: 0;
+}
```

Comment:
Not suggesting a change here, but curious if this is necessary because of a limitation with the `Cell` component?

Reply:
The reason I ask is that I have in the past encountered "surprises" with the `Cell` component being "too clever", altering its layout from perhaps some assumptions of use cases based on props passed. So I'm wondering if you ran into something similar here. :thinking:

Reply:
I'm not sure if I would classify it as "too clever," but for a left-aligned `Cell` if an `icon` is not passed (unselected) then it would "collapse" the space where the icon _would be_ if an icon were passed (selected). So, the end result is that selected and unselected `Cell`s have different left-sided alignment. 

<details><summary>Selected v Unselected Option Alignment</summary>

<img alt="selected-option-alignment" src="https://user-images.githubusercontent.com/438664/134405257-9f9e206f-0b8d-4601-9662-6971fd1ebce5.PNG" width="400" />

</details>

I considered expanding `Cell`'s capabilities to manage this itself, but though it might be premature to add that complexity/conditional to `Cell`. This layout of left-sided checkmark option selection, but also with a right-sided disclosure/value is new and only used here. I thought it might make more see to manage this logic in this one place rather than expand `Cell` now.

Reply:
>  it might be premature to add that complexity/conditional to `Cell`

I agree with this. I think `Cell` is already trying to do too much, imo. Fwiw, your picture jogged my memory on what was surprising: the logic for the separators width was what made assumptions about use cases. In the image you shared above, the "collapsed space" version shows the text component shifted to the left, but the separator remains, so they become misaligned.